spec/features/change_parole_review_date_spec.rb

```diff
@@ -1,46 +1,152 @@
 require 'rails_helper'
 
 RSpec.feature "ChangeParoleReviewDates", type: :feature do
-  # This ID has an indeterminate sentence
-  let(:nomis_offender_id) { 'G0549UO' }
-  let!(:case_info) { create(:case_information, offender: build(:offender, nomis_offender_id: nomis_offender_id)) }
-  let!(:alloc) { create(:allocation_history, prison: prison, nomis_offender_id: nomis_offender_id, primary_pom_nomis_id: 485_926) }
-  let(:year) { Time.zone.today.year + 1 }
-  let(:yesterday) { Time.zone.yesterday }
-  let!(:prison) { 'LEI' }
+  let(:prison) { create(:prison) }
+  let(:tariff_date) { Time.zone.today + 1.year }
+  let(:existing_prd) { nil }
+  let(:nomis_offender_id) { nomis_offender.fetch(:offenderNo) }
+
+  # Stub API response to represent an offender in NOMIS
+  let(:nomis_offender) {
+    build(:nomis_offender, agencyId: prison.code,
+          sentence: attributes_for(:sentence_detail, :indeterminate, tariffDate: tariff_date)
+    )
+  }
```

Comment:
```suggestion
  let(:nomis_offender) do
    build(:nomis_offender, agencyId: prison.code,
          sentence: attributes_for(:sentence_detail, :indeterminate, tariffDate: tariff_date)
    )
  end
```

I tend to prefer using `do`/`end` for multiline blocks.